src/main.c

```diff
@@ -6,6 +6,8 @@
  * Copyright (c) 2012-2017, Andrei Alexeyev <akari@alienslab.net>.
  */
 
+#define SDL_MAIN_HANDLED
```

Comment:
Could you elaborate on this one?

Comment from SDL_main.h:
```
#ifndef SDL_MAIN_HANDLED
#if defined(__WIN32__)
/* On Windows SDL provides WinMain(), which parses the command line and passes
   the arguments to your main function.

   If you provide your own WinMain(), you may define SDL_MAIN_HANDLED
 */
#define SDL_MAIN_AVAILABLE
```

We don't provide our own WinMain, so I'm not sure why would this be needed or how it would even work.

Reply:
I was getting [this error](https://stackoverflow.com/a/25089610/5108318) (mingw inserts its own WinMain which loads main).

Reply:
Interesting. We use the mingw-w64 fork instead of plain mingw, so I'm guessing that one doesn't do it. I wonder if there's some way to make it work on both without user intervention. @lachs0r, any idea?